### ISSUE TITLE
Multiple prompts within one python file

### ISSUE THREAD

**Issue by P-Sood** [promptfoo/promptfoo#507]
Hello, I really enjoy this library, and its helping my team tremendously.

I am wondering if there is any way to test out multiple prompts within one Python file. It would take too long to manually create the proper files and link them, and so we are using Python to format it properly, however we are unable to get 2 columns, where each column represents a different prompt.

```python
def generate_prompt(context: dict, X:list) -> str:
    return (
        f"""
        {X[0]}
        Describe {context['vars']['utterance']} concisely, create only one response from the AI.
---
        {X[1]}
        Describe {context['vars']['utterance']} concisely, create only one response from the AI.
        """
    )

if __name__ == "__main__":

    X = Y("T")

    x1, x2 = X.getx(["T.txt", "T_nof.txt"])

    print(
        generate_prompt(
            json.loads(
                sys.argv[1]
            ),
                [x1, x2],
        )   
    )
```

So as you can see within the code, we get 2 text files, and i want to compare them together. How can i evaluate on both of these prompts within one python script. 

Any help would be appreciated. Thank you :)

**Comment by P-Sood** [comment]
There seems to be a way to do this in Javascript as shown below, the question is how to do it within Python.
"
To reference a specific function in your prompt file, use the following syntax: filename.js:functionName:

```javascript
module.exports.prompt1 = async function ({ vars }) {
  return [
    {
      role: 'system',
      content: `You're an angry pirate. Be concise and stay in character.`,
    },
    {
      role: 'user',
      content: `Tell me about ${vars.topic}`,
    },
  ];
};
```
"

**Comment by typpo** [comment]
#533 adds the requested functionality.  It will be in the next release!

**Comment by typpo** [comment]
Released in 0.46.0
